### DISCUSSION TITLE
Upscaling Images with transparent backgrounds

### DISCUSSION THREAD

**Discussion by Serkunny** [upscayl/upscayl#694] [Q&A] [answered]
Hello,
I was wonder how I can upscale images with transparent background png. My image with a transparent background comes out with a green background instead

**Comment by NayamAmarshe** [comment]
Are you on Windows? Sometimes Windows has that issue. Mac and Linux don't.

**Reply by aaronliu0130** [reply]
#496

**Comment by Serkunny** [comment, marked as answer]
Hello, I have figured it out! I had saved my png as 8 bit, but when I saved it to 32 bit, it worked!

**Comment by ArshCypherZ** [comment]
try https://enhanceai.art/upscaling

**Reply by aaronliu0130** [reply]
Unfortunately not open source
